docs/javascript/v2/get-started/react-sample-app-quickstart.mdx

```diff
@@ -0,0 +1,179 @@
+    4. Add the "Room Id" and role to the localhost URL to test the app. For example, `http://localhost:3000/633fcdd84208780bf665346a/host`
+
+    <br />
+    <video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
+        <source src="/docs/guides/join-room-sample-app.mp4" type="video/mp4" />
+    </video>
+    <br />
+
+## Customize your app
+
+This sample app allows UI customization, like updating the UI layout, logo, tile aspect ratio, theme, color, etc. To make these customizations, you can use various environment variables in the env file.
+
+### Configure UI layout
+
+-   You can customize the layout to arrange the position of video tiles based on roles. The value of `REACT_APP_POLICY_CONFIG` should be a JSON stringified object of the structure:
+
+    ```ts section=configureUi sectionIndex=1
+    type HMSRoleName = string;
+
+    /**
+     * undefined means none
+     * empty array [] for all roles
+     */
+    interface RoleConfig {
+        center?: HMSRoleName[];
+        sidepane?: HMSRoleName[];
+        selfRoleChangeTo?: HMSRoleName[];
+        remoteRoleChangeFor?: HMSRoleName[];
+    }
+
+    interface PolicyConfig {
+        [role: string]: RoleConfig;
+    }
+    ```
+
+-   **Example**: If you wish to keep the video tile of the student in the center for the trainer and his video tile on the side pane, you can use the below JSON stringified object.
+
+    ```bash
+    REACT_APP_POLICY_CONFIG = "{"trainer":{"center": ["student"],"sidepane":["trainer"]}}"
+    ```
+
+### Change logo
+
+You can upload your logo in the sample app to customize the UI for your brand. You can achieve this in two ways:
+
+-   Using the no-code editor in the UI
+
+    <video loop="true" autoplay="autoplay" controls="controls" id="vid" muted>
+        <source src="/docs/guides/logo.mp4" type="video/mp4" />
+    </video>
+    <br />
+
+-   Adding the image URL in `REACT_APP_LOGO` variable in the .env file.
+
+### Change tile aspect ratio
+
+You can use the `REACT_APP_TILE_SHAPE` variable to customize the aspect ratio of video tiles; the format is `width-height`.
+
+**Examples:**
+
+-   `1-1`: square tiles
+-   `4-3`: landscape tiles
+-   `16-9`: wide tiles
+-   `9:16`: mobile view
+
+### Change theme and color
+
+-   Use the `REACT_APP_THEME` variable to switch the theme between dark and light mode
+    -   `dark`: for dark theme
+    -   `light`: for light theme
+-   Use the `REACT_APP_COLOR` variable to customize the UI with your brand colors
+    -   You can give a hex code the color will be used for buttons, border audio level and such.
+
+### Change font
+
+You can use `REACT_APP_FONT` variable to update the font used in the app. The font must be imported in your styling for it to work.
+
+### Enable/disable chat and screenshare
+
+-   You can set the `REACT_APP_SHOW_CHAT` variable as "false" to disable chat
+-   Set `REACT_APP_SHOW_SCREENSHARE` variable as "false" to disable screenshare
```

Comment:
we should talk about how this behaviour overrides the role config

Reply:
removing this as confirmed by @raviteja83 that these are not used.